Derpitcraft/README.md

```diff
@@ -118,50 +78,35 @@ There might be a few compatibility issues with the server, since we support both
         3. You can craft droppers to dispensers. 
         4. You can dye sandstone and universally dye blocks. 
         5. You can make shulkers in any shape. 
-        6. Blackstone can be used everywhere cobblestone is used. 
+        6. Blackstone can be used everywhere cobblestone is used in crafting recipies. 
         7. You get more trapdoors, bark, stairs and bricks upon crafting. 
         8. You can unpack ice, nether wart blocks and wool.
     12. Several other custom commands have been added. These are all listed in the How To Guides
-
 ## How to guides
-
 ### Reporting Players
-
-As a rule of thumb, try to resolve issues peacefully, without moderator intervention. If this doesn't work, report the player by raising a ticket (using `$rt`) and choosing "TrixMC" issue in the select menu. Please include the Player's IGN in your report if you can. It is important to take **screenshots** and even **video recordings** as evidence in some cases. If you feel like your issue is not adequeately solved by the moderators, ping **morpheus** directly.
-
+As a rule of thumb, try to resolve issues peacefully, without moderator intervention. If this doesn't work, report the player by raising a ticket (using `$rt`) and choosing `TrixMC` issue in the select menu. Please include the Player's IGN in your report if you can. It is important to take **screenshots** and even **video recordings** as evidence in some cases. If you feel like your issue is not adequeately solved by the moderators, ping **morpheus** directly.
```

Comment:
quotes make a lot more sense here. Its a given name to the issue type. not really a command or something
```suggestion
As a rule of thumb, try to resolve issues peacefully, without moderator intervention. If this doesn't work, report the player by raising a ticket (using `$rt`) and choosing "TrixMC" issue in the select menu. Please include the Player's IGN in your report if you can. It is important to take **screenshots** and even **video recordings** as evidence in some cases. If you feel like your issue is not adequeately solved by the moderators, ping **morpheus** directly.
```